File: Java/Math/Special Series Sum.java
//https://www.geeksforgeeks.org/problems/special-series-sum0903/1
class Solution {
    static long sumOfTheSeries(long n){
        // code here
        long termSum=0, totalSum=0;
        for(int i=1; i<=n; i++){
            termSum=termSum+i;
            totalSum=totalSum+termSum;
        }
        return totalSum;
    }
}

File: Python/Math/Special Series Sum.py
//https://www.geeksforgeeks.org/problems/special-series-sum0903/1
class Solution:
    def sumOfTheSeries(self, n):
        # code here
        termSum = 0
        totalSum = 0
        for i in range(1, n + 1):
            termSum += i
            totalSum += termSum
        return totalSum
